### PR TITLE
FOSSA workflow added

### PR DESCRIPTION
**Pull request by idvoretskyi**
Fixes https://github.com/servicemeshinterface/smi-spec/issues/183

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

File: .github/workflows/fossa.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.14.x"
+      - run: go version
+      # Runs a set of commands to initialize and analyze with FOSSA
+      - name: run FOSSA analysis
+        env:
+          # FOSSA Push-Only API Token
+          FOSSA_API_KEY: '5cda1cd0d5036f6f368682c23a5fdfec'
```
Comment (stefanprodan) [inline .github/workflows/fossa.yml:21]: I think this should go into the SMI org secrets store in GitHub and referenced here with `${{ secrets. FOSSA_API_KEY }}`

Comment (idvoretskyi) [reply .github/workflows/fossa.yml:21]: @stefanprodan GitHub Actions do not support passing secrets from the forks, and if people will submit PR from their forks (what they typically do), this won't work :( See this as a reference - https://github.com/kedacore/keda/pull/860; we've discovered this issue there.

However, this API key is safe to be exposed publicly (it was designed for this purpose) - https://docs.fossa.com/docs/api-reference#push-only-api-token

**Comment by idvoretskyi** [comment]
It's unclear to me now what causes an issue - https://github.com/servicemeshinterface/smi-sdk-go/pull/86/checks?check_run_id=976150974#step:5:22; in touch with the FOSSA team on debugging.

File: .github/workflows/fossa.yml (hunk 1 above)
Comment (stefanprodan) [reply .github/workflows/fossa.yml:21]: Ah I totally forgot about that GH limitation...

**Comment by idvoretskyi** [comment]
@stefanprodan PTAL, looks good now.

**Comment by stefanprodan** [comment]
The URL to the report returns a 404

**Comment by idvoretskyi** [comment]
@stefanprodan fixed, thanks for pointing out.

**Comment by stefanprodan** [comment]
Screenshot 2020-08-14 at 10 22 45

@idvoretskyi I can't access the report even if I'm a maintainer of the SDK and I did login with GitHub.

**Comment by idvoretskyi** [comment]
@stefanprodan if you'll scroll down, you'll see the full public report.

In addition, I'm trying to invite you and other project maintainers to the FOSSA UI to get the maintainers access, but this part of UI doesn't seem to be working (reported to FOSSA).

However, the full report is already r/o available so I think we can proceed with merging this now. I'll work on granting you the full r/w access separately.

**Comment by idvoretskyi** [comment]
@stefanprodan your access should be fixed now, please check. If all good, I'll proceed with adding other project maintainers to the org.

**Comment by idvoretskyi** [comment]
@stefanprodan anything else is needed to get this merged?

**Comment by idvoretskyi** [comment]
Ref - https://github.com/cncf/foundation/issues/109

**Comment by stefanprodan** [comment]
Hey @idvoretskyi I was on PTO, I'll merge the FOSSA PRs now, look ok thank you!

**Comment by idvoretskyi** [comment]
@stefanprodan thanks!

**Pull request merged**